pom.xml

```diff
@@ -228,6 +228,14 @@
     <unitTests.dynamicFactor>1</unitTests.dynamicFactor>
   </properties>
   <repositories>
+    <repository>
+      <id>spring-milestones</id>
+      <name>Spring Milestones</name>
+      <url>https://repo.spring.io/milestone</url>
+      <snapshots>
+        <enabled>false</enabled>
+      </snapshots>
+    </repository>
```

Comment:
Should central be kept as the first repository in the list? That was part of optimizations for maven build (cc @matteo4diani)

Reply:
I will remove the Spring Milestones repository since we are no longer using Spring RCs.